packages/validator/src/commands/validateCmd.ts

```diff
@@ -1,23 +1,39 @@
+import yargs from 'yargs';
+import { hideBin } from 'yargs/helpers';
 import * as utils from './utils';
 import * as logger from '../utils/logger';
 import { validate } from '../validator';
 import { Log, templates } from '../types';
 
-let template = process.env.npm_config_template || process.argv[2];
-const specs = process.env.npm_config_specs || process.argv[3];
-const version = process.env.npm_config_templateVersion || process.argv[4];
-
 const messages: Log[] = [];
 
+const args = yargs(hideBin(process.argv))
+  .option('template', {
+    description: 'Path to validator template file or name of airnode specification format',
+    alias: 't',
+    type: 'string',
+    demandOption: true,
+  })
+  .option('specification', {
+    description: 'Path to specification file that will be validated',
+    alias: ['specs', 's'],
+    type: 'string',
+    demandOption: true,
+  })
+  .parseSync();
+
+// eslint-disable-next-line prefer-const
+let [template, version] = args.template.split('@');
+
 if (template) {
   if (templates[template.toLowerCase() as keyof typeof templates]) {
     template = utils.getPath(templates[template.toLowerCase() as keyof typeof templates], messages, version);
   } else if (version) {
     messages.push(logger.warn('Version argument will be ignored when validating provided template file'));
   }
 
-  if (specs) {
-    const res = validate(specs, template);
+  if (args['specification']) {
```

Comment:
You're again accessing `args` with both `.` and `[]` operator for no apparent reason.

Reply:
Yeah I missed that one somehow, fixed in ffaa558e18addc5796605e7ec9706ebaeb278beb